### DISCUSSION TITLE
SUBSCRIBE/NOTIFY BLF handling

### DISCUSSION THREAD

**Discussion by demonspork** [sipcapture/homer#637] [Q&A] [answered]
How do I get homer to show SUBSCRIBE/Notify packets in a search? I find myself needing to debug some BLF issues with phones using SUBSCRIBE/NOTIFY Messages but I can't figure out how to get homer to search and display those messages.

This sounds like it should be fairly simple and I'm 99% sure I got it working years ago when I last used Homer, but I am finding disappointingly little documentation on how to actually accomplish this task. I get calls and registrations all day, but that was available out of the box without any intervention on my part beyond installing.

**Comment by hamidrhashmi** [comment, marked as answer]
try adding Widget Proto Search

![image](https://github.com/sipcapture/homer/assets/14875110/b952d080-05ac-4cfa-83a9-67da80e2ca88)

It will show you the other Methods Like NOTIFY/PUBLISH/SUBSCRIBE etc.
